build/rewrite.go

```diff
@@ -450,7 +456,7 @@ func sortStringList(x Expr, context string) {
 		return
 	}
 
-	forceSort := keepSorted(list) || keepSorted(list.List[0])
+	forceSort := forceKeepSorted(list) || forceKeepSorted(list.List[0])
```

Comment:
this bit of logic requires users to explicitly comment "keep sorted" on lists with comments